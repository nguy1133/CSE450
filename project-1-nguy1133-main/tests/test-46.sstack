// ERROR: POP can have only one argument.
PUSH "helper..."
POP abc def
