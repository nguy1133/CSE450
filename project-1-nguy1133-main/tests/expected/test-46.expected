ERROR (line 3): Unexpected token 'def'
